docs/switching_from/conda.md

```diff
@@ -0,0 +1,83 @@
+# Transitioning from the `conda` or `mamba` to `pixi`
+Welcome to the guide designed to ease your transition from `conda` or `mamba` to `pixi`.
+This document compares key commands and concepts between these tools, highlighting `pixi`'s unique approach to managing environments and packages.
+With `pixi`, you'll experience a project-based workflow, enhancing your development process, and allowing for easy sharing of your work.
+
+## Why Pixi?
+
+`Pixi` builds upon the foundation of the conda ecosystem, introducing a project-centric approach rather than focusing solely on environments.
+This shift towards projects offers a more organized and efficient way to manage dependencies and run code, tailored to modern development practices.
```

Comment:
Maybe specify that the dependency management is automated rather than manual?
```suggestion
This shift towards projects offers a more automated, organized, and efficient way to manage dependencies and run code, tailored to modern development practices.
```